.travis.yml

```diff
@@ -5,11 +5,10 @@ services:
 
 before_script:
   - docker-compose -f docker-compose.yml -f docker-compose.travis.yml up -d
+  - docker-compose ps
   - docker-compose exec web sh -c "apk add --no-cache git"
-  - docker-compose exec web sh -c "pipenv install -d tox"
+  - docker-compose exec web sh -c "pipenv install tox"
+
 
 script:
   - docker-compose exec web sh -c "pipenv run tox"
-
-after_script:
-  - docker-compose -f docker-compose.yml -f docker-compose.travis.yml down
```

Comment:
Is the removal of this line necessary? I'd like to ensure that all resources for the job are truly cleaned up once the tests are executed...

Reply:
I removed it when I was debugging why the build errored out, its my mistake the line has been added back to the yaml file.